src/psyclone/tests/domain/nemo/transformations/nemo_arrayrange2loop_trans_test.py

```diff
@@ -486,14 +581,12 @@ def test_apply_imported_function():
     # functions or arrays always as arrays accessors, for this reason the error
     # message talks about arrays instead of functions. If this is resolved this
     # test would be equivalent to test_apply_with_a_function_call.
-    assert("Error in NemoArrayRange2LoopTrans transformation. This "
-           "transformation does not support assignments with rhs arrays that "
-           "don't have a range, but found 'ptr_sjk' in:\n"
-           "z3d(1,:,:) = ptr_sjk(pvtr(:,:,:),btmsk(:,:,jn) * btm30(:,:))\n"
-           in str(info.value))
+    assert ("Error in NemoArrayRange2LoopTrans transformation. This "
+            "transformation does not support array assignments that contain "
+            "nested Range structures, but found:\n" in str(info.value))
 
 
-def test_apply_calls_validate():
+def test_validate_calls_validate():
```

Comment:
I think this one should still be called `test_apply_calls_validate` :-)

Reply:
Good catch ;) Fixed